metpy/plots/skewt.py

```diff
@@ -416,6 +416,32 @@ def plot_barbs(self, p, u, v, c=None, xloc=1.0, x_clip_radius=0.1,
         b.set_clip_box(transforms.TransformedBbox(ax_bbox, self.ax.transAxes))
         return b
 
+    def get_t0(self, spacing=10):
```

Comment:
Can we call this something like `_get_line_starting_temps`? I'd like to not have it a public method and I think this makes it clearer to me what it's for.

Reply:
Sure thing, I'll set this up when I'm free for an evening and try to get a lint free, tested commit with enforced units for X and Y axes.

I found that using Pascals caused hard to fix (for me) problems in the plotting, how do I enforce not only that units exist but that they are mbar or hPa? or should I just transform Y units in the __init__ and subroutines?

Reply:
Can you detail what problems you're running into with Pascals? I want to make sure we understand what the problem is before settling on a solution.

Reply:
I'll test again after temperature units are handled I think, since a few things have changed now.

I'm currently unable to run tests on the latest iteration:
`(devel) latitude:MetPy$ pytest
ImportError while loading conftest '/home/jesse/Desktop/Repos/MetPy/conftest.py'.
conftest.py:15: in <module>
    import metpy.calc
E   ModuleNotFoundError: No module named 'metpy'
`
Do i need to update my devel environment somehow since the project refactoring?

Reply:
Yeah I think I had to do a new `pip install -e .` in the `MetPy` directory so that everything is redirected through `src`. Thanks for your work on this!